src/appleseed/renderer/modeling/bsdf/metalbrdf.cpp

```diff
@@ -299,25 +297,17 @@ namespace
         {
             if (values->m_energy_compensation != 0.0f)
             {
-                const float Ess =
-                    get_directional_albedo(
-                        std::abs(dot(outgoing, n)),
-                        values->m_roughness);
+                const float Ess = get_directional_albedo(
```

Comment:
`ess` (lower case)

What does it stand for?

Reply:
This comes from the original paper. E is for energy and ss for single scattering.